### ISSUE TITLE
🐞 - Show arrow option not reflected when set through setDefaults

### ISSUE THREAD

**Issue by danipavic** [hakimio/ngx-ui-tour#180]
### Which ngx-ui-tour-* package is the source of the bug?

md-menu

### Description

If I want to disable the arrow on all steps I would like to set this as a default option, however it is not working properly.

With the following code below, the arrow is present on all steps even though it shouldn't be.

```
this.tourService.setDefaults({
	enableBackdrop: true,
	showArrow: false,
	popoverClass: 'question-grid-step-popover',
});
```

If I add the showArrow flag to a step option explicitly, then it works.

```
this.tourService.initialize([
	{
		anchorId: OnboardingStep.Timer,
		content: this.translocoService.translate('onboarding.timerStep'),
		showArrow: false,
	},
]);
```

### Angular version

17.3.9

### Ngx UI Tour version

12.0.1

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

**Comment by hakimio** [comment]
You can set the defaults in `initialize()` method until this is fixed: `initialize(steps, defaults)`.

**Comment by hakimio** [comment]
Fixed in [v12.1.0](https://github.com/hakimio/ngx-ui-tour/releases/tag/v12.1.0).
